docs/developer/decision-records/2022-02-11-codeql/codeql.md

```diff
@@ -0,0 +1,45 @@
+# CodeQL
```

Comment:
if you name this file README.md it will be displayed on navigation in GitHub

Reply:
done